src/app/public/modules/href/href-resolver.service.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { TestBed } from '@angular/core/testing';
+import { HrefResolverFixtureService } from './fixtures/href-resolver-fixture.service';
+import { SkyHrefResolverService } from './href-resolver.service';
+
+describe('HREF Resolver Service', () => {
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [
```

Comment:
Since you're not testing a component that injects `SkyHrefResolverService` or using `TestBed.inject()` to get an instance of `SkyHrefResolverService`, you don't need to provide them in the test bed configuration. This entire `beforeEach()` can therefore be removed.